definitions/variable/emissions/emissions.yaml

```diff
@@ -43,52 +75,118 @@
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+- Emissions|{Level-3 Species}|AFOLU|Biomass Burning:
+    description: Emissions from agricultural waste burning
+    unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land:
-    description: Emissions of {Level-3 Species} from forestry and other land use
+    description: Emissions of {Level-3 Species} from forestry and other land use and 
+      land use change. Removals in this variable include agroforestry, re/afforestation, 
+      biochar, soil carbon management and forest management.
+    unit: "{Level-3 Species}"
+    notes: Emissions from land use change are reported under the land use type
+      that the land is converted to. For instance, deforestation for the creation
+      of new cropland is reported under "Emissions|*|AFOLU|Land|Cropland".
+    components:
+      - Emissions|{Level-3 Species}|AFOLU|Land|Forest Land
+      - Emissions|{Level-3 Species}|AFOLU|Land|Cropland
+      - Emissions|{Level-3 Species}|AFOLU|Land|Grassland
+      - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands
+      - Emissions|{Level-3 Species}|AFOLU|Land|Settlements
+      - Emissions|{Level-3 Species}|AFOLU|Land|Other Land
+      - Emissions|{Level-3 Species}|AFOLU|Land|Fires
+- Emissions|{Level-3 Species}|AFOLU|Land|Forest Land:
+    description: Emissions and removals from lands with woody vegetation
+      (partially IPCC 1996 category 5A/5B/5D; IPCC 2006 category 3B1) 
+    notes: It also includes systems with vegetation that currently fall below, 
+      but are expected to later exceed, the threshold values used by a country to
+      define the forest land category.
+    unit: "{Level-3 Species}"
+- Emissions|{Level-3 Species}|AFOLU|Land|Cropland:
+    description: Emissions and removals from arable and tillage land, rice
+      fields, and agro-forestry systems where vegetation falls
+      below the thresholds used for the forest land category
+      (no clear IPCC 1996 category; IPCC 2006 category 3B2)
+    notes: This does not include methane emissions from rice cultivation
```

Comment:
Why is CH4 from rice cultivation excluded? It also contradicts with the desc: "rice fields"